web/src/components/storage/ProposalPageOptions.jsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) [2023] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React, { useEffect, useState } from "react";
+import { useHref } from "react-router-dom";
+import { useInstallerClient } from "~/context/installer";
+import { If, ContextualActions } from "~/components/core";
+
+const DASDLink = () => {
+  const href = useHref("/storage/dasd");
+
+  return (
+    <ContextualActions.Item
+      key="dasd-link"
+      href={href}
+      description="Manage and format"
+    >
+      DASD
+    </ContextualActions.Item>
+  );
+};
+
+const ISCSILink = () => {
+  const href = useHref("/storage/iscsi");
+
+  return (
+    <ContextualActions.Item
+      key="iscsi-link"
+      href={href}
+      description="Connect to iSCSI targets"
+    >
+      iSCSI
+    </ContextualActions.Item>
+  );
+};
+
+export default function ProposalPageOptions () {
+  const [showDasdLink, setShowDasdLink] = useState(false);
+  const { storage: client } = useInstallerClient();
+
+  useEffect(() => {
+    client.dasd.isSupported().then(setShowDasdLink);
+  }, [client.dasd]);
+
+  return (
+    <ContextualActions>
+      <ContextualActions.Group
+        label="Configure"
```

Comment:
Instead of having a lot of "similar actions" like "Configure DASD", "Configure iSCSI", and "Configure FCoE" for example, we decided to group them under the "Configure" group. Now can be difficult to see the benefit of grouping page options, but think that most probably it will grow sooner than later. Anyway we can re-evalute it if needed an introduce groups later, but please keep in mind that its a bit confusing having a bunch of "Configure ACRONYM" options in a row.